microsoftgraph/msgraph-cli-archived
# Feat/rename command groups

## Overview

Fixes #70 
Pages long outputs

### Demo

N/A

### Notes

This PR does not rename as many commands as we would like to have renamed. To do that I will need to know what commands to rename to what.


## Testing Instructions

* Pull these changes
* Observe that you can create an application with `./mg applications application create`
* Observe that you can create users with `./mg users user create`
* Run `./mg users user create -h` 
* Observe that the output is paged
